css/styles.css

```diff
@@ -497,10 +497,6 @@ section#developers .section-header {
 
```

Comment:
I can't really say. While I understand the need to 1. have two independent docs and 2. copied articles, I fear that the whole thing will explode on you at some point :O
The building process seems a bit unclear right now. There is no automatic build after PR merge, it is hard to understand what is copied and why there are duplicate files (Contribution for example).

Would it be better to have the whole eclise smarthome docs as a submodule (to make realtions clear to everyone) and to work with symlinks (if jekyll and friends like that)? Without symlink an idea would be to have an sh script (like you have now) copying files with an additional step, marking them as copied. An added YAML front matter "page.source" in the head of the file would be a nice approach I guess.